martin/src/utils/id_resolver.rs

```diff
@@ -122,16 +134,22 @@ mod tests {
 
     #[test]
     fn id_resolve() {
-        let r = IdResolver::default();
+        let reserved = ["foo"];
```

Comment:
let's use a more descriptive one here

```suggestion
        let reserved = ["reserved"];
```